### PR TITLE
Add List and cardview components for ingest sheets list

### PR DESCRIPTION
**Pull request by katdivyareddy10**
Created components for switching between list view and card view.
Added toggle to switch between the views.

Screen Shot 2020-04-29 at 5 10 24 PM
Screen Shot 2020-04-29 at 5 10 50 PM
Screen Shot 2020-04-29 at 5 10 58 PM

### CODE REVIEW

File: assets/js/components/IngestSheet/Completed.jsx [hunk 1]
```diff
@@ -45,17 +51,37 @@ const IngestSheetCompleted = ({ sheetId }) => {
...
+              key={work.id}
+              className="column is-half-tablet is-one-quarter-desktop"
+            >
+              <WorkCardItem key={work.id} work={work} />
+            </div>
+          ))}
+        </div>
+      )}
+      {isListView && (
+        <div className="">
```
Comment (adamjarling) [inline assets/js/components/IngestSheet/Completed.jsx:77]: If an empty `div`, let's use the React empty div `<> ... </>` to minimize extra markup that gets rendered

File: assets/js/components/Work/UIWorkCardItem.jsx [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+              onError={(e) => {
+                e.target.src = "/images/1280x960.png";
+              }}
+            />
+          </Link>
+        </figure>
+      </div>
+      <div className="card-content">
+        <h3 className="title is-size-4">
+          {work.accessionNumber} <span className="tag">{work.workType}</span>
```
Comment (adamjarling) [inline assets/js/components/Work/UIWorkCardItem.jsx:27]: Let's display a Work title (located in DescriptiveMetadata), by default if one exists, and "Untitled" as a fallback.  Then let's move Accession Number into the `<dd>` list I mentioned in the screen shot notes.

File: assets/js/components/Work/UIWorkCardItem.jsx [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+          <span
+            data-testid="tag-visibility"
+            className={`tag ${setVisibilityClass(work.visibility)}`}
+          >
+            {work.visibility.toUpperCase()}
+          </span>
+        </h3>
+
+        {/* <p className="subtitle is-size-6">Accession Number</p> */}
+        <h4 className="subtitle">
```
Comment (adamjarling) [inline assets/js/components/Work/UIWorkCardItem.jsx:37]: All this is replaced by the `<dd>` as mentioned in the notes.

File: assets/js/components/Work/UIWorkListItem.jsx [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+        data-testid="ui-worklist-item"
+      >
+        <figure className="media-left">
+          <p className="image is-128x128">
+            <Link to={`/work/${work.id}`}>
+              <img
+                src={`${work.representativeImage}/full/1280,960/0/default.jpg`}
+                alt={work.title}
+                data-testid="image-work"
+                onError={(e) => {
```
Comment (adamjarling) [inline assets/js/components/Work/UIWorkListItem.jsx:22]: I'm not sure `onError()` will ever get hit, and if it does its by accident.   Let's move this conditional logic into the `scr` attribute above.

File: assets/js/components/Work/UIWorkListItem.jsx [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+                  e.target.src = "/images/1280x960.png";
+                }}
+              />
+            </Link>
+          </p>
+        </figure>
+        <div className="media-content">
+          <div className="content">
+            <h3 className="title is-size-4">
```
Comment (adamjarling) [inline assets/js/components/Work/UIWorkListItem.jsx:31]: See comments above and please make the same changes here.  Thanks!

**Comment by adamjarling** [comment]
For the row layout, see below:

image

On the table layout, if you wrap the `table` in a `<div className="table-container">` parent element, Bulma will make the table scrollable horizontally (in case screen width gets small, but not quite `touch` size.

Any questions, lemme know.  Hopefully these should be pretty quick fixes, then this is ready to merge.

**Comment by katdivyareddy10** [comment]
UPDATED IMAGES: Screen Shot 2020-05-04 at 10 33 09 AM

**Comment by katdivyareddy10** [comment]
Switched toggle to show both view buttons per @adamjarling  comments

Screen Shot 2020-05-04 at 4 48 24 PM

**Pull request merged**